libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3717,6 +3728,11 @@ MAV_RESULT GCS_MAVLINK::handle_command_do_set_home(const mavlink_command_long_t
     return MAV_RESULT_ACCEPTED;
 }
 
+MAV_RESULT GCS_MAVLINK::handle_command_do_scale_with_zoom(const mavlink_command_long_t &packet)
```

Comment:
This handler's definition should actually be implemented in the `GCS_COPTER` derived class, such that it can access the `scale_with_zoom` parameter of the `Copter` class.